frontend/packages/core/src/AppProvider/index.tsx

```diff
@@ -91,29 +91,35 @@ const ClutchApp: React.FC<ClutchAppProps> = ({
         <div id="App">
           <ApplicationContext.Provider value={{ workflows: discoverableWorkflows }}>
             <Routes>
-              <Route path="/*" element={<AppLayout isLoading={isLoading} />}>
-                <Route key="landing" path="/" element={<Landing />} />
+              <Route path="/" element={<AppLayout isLoading={isLoading} />}>
+                <Route path="/" element={<Landing />} />
                 {workflows.map((workflow: Workflow) => (
-                  <ErrorBoundary workflow={workflow} key={workflow.path.split("/")[0]}>
```

Comment:
Moving this inside of the element seems to work fine locally. Was looking through this guide: https://reactrouter.com/docs/en/v6/upgrading/v5

Reply:
Seems ok. My only concern would be if react-router is failing would the error boundary still capture it?

Reply:
So looking into ErrorBoundaries, it looks like it will catch a thrown error from the closest wrapped component. So ours only works for the workflow (which is why its a required prop) right? I verified that it will catch errors from the workflows the same.

I tried throwing an error below the outlet under the routes map (to try and simulate a router error) and it seems to behave the same as main, in that it doesn't catch it and fails.

Does that sound about right as far as current working conditions?

Reply:
error boundaries should catch error anywhere in the child component tree (e.g. previously if `Route` had failed to render it would have been captured by the error boundary). Errors thrown in the error boundary are not caught, which I think this would fall into. It's likely not a huge concern as the odds of the route having issues and not being caught before a deploy is low but I'm curious if there is a reason to change the behavior?

Reply:
It throws an error with this branch if we leave it was the only reason I started looking into it:

"index.tsx:19 Uncaught Error: [ErrorBoundary] is not a <Route> component. All component children of <Routes> must be a <Route> or <React.Fragment>"

Reply:
:ok_hand: